### DISCUSSION TITLE
BRouter external profile options

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#621] [Q&A] [answered]
Loading external BRouter profiles into Cruiser will now offer all profile options.

External profile options are automatically generated from the profile file.
It works exactly as on the [BRouter website](https://brouter.de/brouter-web/).

<img src="https://github.com/user-attachments/assets/8d164291-c8a8-458e-9f68-1e7d6898ee19" width="50%"/><img src="https://github.com/user-attachments/assets/4034363c-93e4-4a89-b414-be19e849d7bc" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
- Loading external BRouter profiles offers all profile options in https://github.com/devemux86/cruiser/discussions/641

**Comment by Leeb64** [comment]
Changing the profile options works - the first time.
The changed values ​​are adopted.Good.
If I want to change other values ​​later, these will be adopted, but the values ​​changed the first time will be reset to the original value.

**Reply by devemux86** [reply]
@Leeb64 What do you mean later? What are the exact steps?

I just tested it again:
- Load an external BRouter profile
- Change some options
- Restart the app
- The changed options remain

**Reply by Leeb64** [reply]
-Load an external BRouter profile
-  Change some options
 - Restart the app
 - The changed options remain
---------------------------
-Now change further options.
-Calculate the route
-The options that were changed first are reset.

**Reply by devemux86** [reply]
@Leeb64 Thanks for the detailed steps.

This should be fixed in **Cruiser 5.0.2-b6**.

**Reply by devemux86** [reply]
- Fixed in https://github.com/devemux86/cruiser/discussions/641

**Comment by DolomitesTom** [comment]
Can you please provide a default external brouter file?

**Reply by devemux86** [reply]
- Cruiser driving profile is here: https://github.com/devemux86/cruiser/discussions/272

- All BRouter profiles are [here](https://github.com/abrensch/brouter/tree/master/misc/profiles2)

- All @poutnikl profiles are [here](https://github.com/poutnikl/Brouter-profiles/tree/master)

**Reply by DolomitesTom** [reply]
Thank you. Is it possible to create one from the actual settings in Cruiser?

**Reply by devemux86** [reply]
The reverse process is the expected one, we load an external BRouter profile and configure its options in Cruiser.

External BRouter profiles can be further edited using a text editor and tested on the BRouter website.
